src/app/home/home.style.scss

```diff
@@ -0,0 +1,63 @@
+.home {
+
+    .nav {
+        height: 70px;
+
+        .create {
```

Comment:
please avoid cascades as possible, it is highly not recommended to use cascades with bem